platform/drivers/baseband/radio_MD3x0.cpp

```diff
@@ -231,6 +232,10 @@ bool radio_checkRxDigitalSquelch()
 
 void radio_enableRx()
 {
+    if(config->opMode == OPMODE_DMR) {
```

Comment:
Please don't assign the DMR ID inside the radio driver. Instad, do it in the OpMode handler.
The rationale is that this driver is meant to handle only the low-level management of the radio part itself, like PLL and signal path, while all the protocol-specific stuff is handled by each OpMode handler.